### ISSUE TITLE
How to set ‘ignore_label’？

### ISSUE THREAD

**Issue by QL111111** [jianlong-yuan/LovaszSoftmax_tf#2]
There is a “ignore” parameter in your code. 
`lovasz_loss=lovasz_softmax(tf.nn.softmax(logits),labels,classes='all',per_image=False,ignore=ignore_label)`
How do I set this parameter?

**Comment by jianlong-yuan** [comment]
When the image is croped, it need be padded. The padding area is the ignore label.
